src/db_iter.h

```diff
@@ -154,7 +164,8 @@ class TitanDBIterator : public Iterator {
     }
 
     buffer_.Reset();
-    s = it->second->Get(options_, index.blob_handle, &record_, &buffer_);
+    OwnedSlice blob;
+    s = it->second->Get(options_, index.blob_handle, &record_, &blob);
```

Comment:
Ideally when blob_cache is not enabled, then we can still reuse the buffer_ instead of have a new OwnedSlice.